cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1 on 2024-12-16 06:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="movie",
+            name="image",
+            field=models.ImageField(null=True, upload_to="???"),
```

Comment:
The `upload_to` parameter in `models.ImageField` is set to `"???"`. You need to specify a valid directory path where the uploaded images should be stored. For example, you could use `upload_to='images/'` to store images in an `images` directory.